shopno/index.html

```diff
@@ -0,0 +1,586 @@
+                            We Actually Do amazing works
+                        </h3>
+                        <p class="about__content-block-text">
+                            A visual approach is an approach to a runway at an airport 
+                            conducted under instrument flight rules. The pilot must at all times have either the airport or the preceding aircraft in approach correct sight. conducted under instrument flight rules. The pilot must at all times.
+                            <br>
+                            <br>
+                            Preceding aircraft in approach correct sight. conducted under instrument flight rules.
+                        </p>
+                        <h3 class="about__content-block-title">
+                            why we are the great?
+                        </h3>
+                        <p class="about__content-block-text">
+                            A visual approach is an approach to a runway at an airport 
+conducted under instrument flight rules.<br>
+The preceding aircraft in approach correct sight.
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="works">
+            <div class="works__container container">
+                <h2 class="works__title title">
+                    Featured works
+                </h2>
+                <p class="works__title-addition title-addition">
+                    Lorem ipsum dolor sit amet event landing template
+                </p>
+                <ul class="works__examples examples" id="examplesBlock">
+                    <li class="examples__item">
+                        <picture class="examples__picture">
+                            <source
+                                media="(min-width: 1024px)"
+                                srcset="./src/images/work/work-1-370.webp"
+                                type="image/webp"
+                            >
+                            <source
+                                media="(max-width: 1024px) and (min-width: 375px)"
+                                srcset="./src/images/work/work-1-452.webp"
+                                type="image/webp"
+                            >
+                            <source
+                                media="(max-width: 375px)"
+                                srcset="./src/images/work/work-1-256.webp"
+                                type="image/webp"
+                            >
+                            <source
+                                media="((max-width: 1024px) and (min-width: 375px))"
+                                srcset="./src/images/work/work-1-452.jpg"
+                            >
+                            <source
+                                media="(max-width: 375px)"
+                                srcset="./src/images/work/work-1-256.jpg"
+                            >
```

Comment:
Why you don't use `srcset` here and beyond?

Reply:
In ```srcset``` i can't use different types image-formats. In this case I am trying use ```webp``` and ```jpg```.